pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator_helper.go

```diff
@@ -32,13 +33,19 @@ import (
 func (c *K8sOrchestrator) getPVCAnnotations(ctx context.Context, volumeID string) (map[string]string, error) {
 	log := logger.GetLogger(ctx)
 	log.Debugf("Getting annotations on pvc corresponding to volume: %s", volumeID)
+	pvcAnn := make(map[string]string)
```

Comment:
The doc for the method is not misleading since we are returning empty map if the PVC is not found. See my other suggestion.